src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -339,7 +339,8 @@
 									x:Name="Column10"
 									Width="{x:Bind ColumnsViewModel.StatusColumn.Length, Mode=OneWay}"
 									MaxWidth="{x:Bind ColumnsViewModel.StatusColumn.MaxLength, Mode=OneWay}" />
-								<ColumnDefinition x:Name="Column11" Width="0" />
+								<ColumnDefinition Width="Auto" />
```

Comment:
This doesn't seem correct, does the size get saved to the database?

Reply:
Yes it does. That line defines the grid splitter column

Reply:
Is the column supposed to have a name?

Reply:
I'll remove it since it's not needed